udata/templates/dataset/display.html

```diff
@@ -327,9 +329,11 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
                         </div>
 
                         {% for key, value in dataset.extras.items() %}
-                        <div itemprop="additionalProperty" itemscope itemtype="http://schema.org/PropertyValue">
-                            <meta itemprop="name" content="{{key}}"/>
-                            <meta itemprop="value" content="{{value.serialize()|to_json if value.serialize else value }}"/>
+                        <div itemscope itemtype="http://schema.org/PropertyValue">
+                            <div itemprop="additionalProperty">
```

Comment:
I think the `additionnalProperty` and its div are not needed anymore. We only want to have `name` and `value` defined on `PropertyValue`